src/i18n/strings/en_EN.json

```diff
@@ -2422,6 +2422,24 @@
         "enable_markdown": "Enable Markdown",
         "enable_markdown_description": "Start messages with <code>/plain</code> to send without markdown.",
         "encryption": {
+            "advanced": {
+                "breadcrumb_first_description": "Your account details, contacts, preferences, and chat list will be kept",
+                "breadcrumb_page": "Reset encryption",
+                "breadcrumb_second_description": "You will lose any message history that’s stored only on the server",
+                "breadcrumb_third_description": "You will need to verify all your existing devices and contacts again",
```

Comment:
I think this could be a bit misleading, as it says that the user needs to verify all their contacts, whereas they really only need to re-verify contacts that they had already verified.  Though I don't know how to word it better without being overly wordy.  So maybe it's OK.

Reply:
The wording is coming from [figma design](https://www.figma.com/design/qTWRfItpO3RdCjnTKPu4mL/Settings?node-id=380-87579&t=OUJfK2vXKDjIWapb-4)